lib/azure/armrest/model/base_model.rb

```diff
@@ -189,7 +190,9 @@ class ResourceProvider < BaseModel; end
     class Sku < BaseModel; end
     class Usage < BaseModel; end
 
-    class ResponseHeaders < BaseModel; end
+    class ResponseHeaders < BaseModel
+      undef_method :response_headers
```

Comment:
@bzwei I just thought it was inappropriate for a ResponseHeaders object to have a :response_headers method. It was actually causing some confusion for me in development.

If there's a nicer way, I'm listening.